malasakit-django/pcari/migrations/0049_auto_20170530_2213.py

```diff
@@ -0,0 +1,115 @@
+        ),
+        migrations.AlterField(
+            model_name='generalsetting',
+            name='english_more_info',
+            field=models.CharField(default='More Information', max_length=30),
+        ),
+        migrations.AlterField(
+            model_name='generalsetting',
+            name='english_scale_description',
+            field=models.CharField(default='0 (strongly disagree) to 9 (strongly agree)', max_length=300),
+        ),
+        migrations.AlterField(
+            model_name='generalsetting',
+            name='english_select',
+            field=models.CharField(default='Select', max_length=30),
+        ),
+        migrations.AlterField(
+            model_name='generalsetting',
+            name='english_short_description',
+            field=models.CharField(default='A project by the CITRIS Connected Communities Initiative at UC Berkeley and the Philippine Commission on Higher Education through the Philippine-California Advanced Research Institutes Project.', max_length=300),
+        ),
+        migrations.AlterField(
+            model_name='generalsetting',
+            name='filipino_about_pcari',
+            field=models.CharField(default='Tungkol sa PCARI', max_length=30),
+        ),
+        migrations.AlterField(
+            model_name='generalsetting',
+            name='filipino_exit',
+            field=models.CharField(default='Lumabas', max_length=30),
+        ),
+        migrations.AlterField(
+            model_name='generalsetting',
+            name='filipino_female',
+            field=models.CharField(default='Babae', max_length=30),
+        ),
+        migrations.AlterField(
+            model_name='generalsetting',
+            name='filipino_learn_more',
+            field=models.CharField(default='Alamin ang iba pang impormasyon kung paano magiging handa sa isang sakuna', max_length=300),
+        ),
+        migrations.AlterField(
+            model_name='generalsetting',
+            name='filipino_male',
+            field=models.CharField(default='Lalaki', max_length=30),
+        ),
+        migrations.AlterField(
+            model_name='generalsetting',
+            name='filipino_more_info',
+            field=models.CharField(default='Iba pang impormasyon', max_length=30),
+        ),
+        migrations.AlterField(
+            model_name='generalsetting',
+            name='filipino_scale_description',
+            field=models.CharField(default='Mula 0 (hinding-hindi ako sumasang-ayon) hanggang 9 (lubos akong sumasang-ayon)', max_length=300),
+        ),
+        migrations.AlterField(
+            model_name='generalsetting',
+            name='filipino_short_description',
+            field=models.CharField(default='Isang proyekto ng CITRIS Connected Communities Initiative ng UC Berkeley, at ng Commission on Higher Education ng Pilipinas sa pamamagitan ng Philippine-California Advanced Research Institutes Project', max_length=300),
+        ),
+    ]
```

Comment:
Didn't spot this, but we don't need this migration, since `GeneralSetting` is deprecated.